### PR TITLE
Clement remove bold font weight everywhere

### PR DESCRIPTION
**Pull request by Bookie0**
fixes [this](https://www.notion.so/Idol-Lead-Sync-781de97ad403492b94b420349f829ea5?pvs=4#1550ad723ce1804f8cb6ccc71c673a54)

removes font weight bold
sorry no time for screenshots 🙃

### CODE REVIEW

File: new-dti-website/components/initiatives/InitiativeDisplay.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const Initiative = ({
           width >= LAPTOP_BREAKPOINT ? inter.className : ibm_plex_mono.className
         }`}
       >
-        {parts.map((part, index) => (part === eventName ? <b key={index}>{part}</b> : part))}
+        {parts.map((part, index) => part)}
```
Comment (andrew032011) [inline new-dti-website/components/initiatives/InitiativeDisplay.tsx:52]: Wait can't we just do parts here then?

File: new-dti-website/components/initiatives/data/initiatives.json [hunk 2]
```diff
@@ -15,8 +15,7 @@
       "height": 328
     },
     "description": "At the Tompkins County Public Library, our members give children in the Ithaca community the opportunity to grow and learn about other technical careers. We introduce code and design through beginner-friendly, interactive workshops with Cornell students, providing individualized feedback. Young students can learn Scratch to create their own games and animations, and Figma to make eye-catching visuals with shapes, colors, and more!",
-    "mobileDescription": "@Tompkins County Public Library, we introduced code & design with workshops that taught Figma and Sketch.",
-    "eventName": "Tompkins County Public Library"
```
Comment (andrew032011) [inline new-dti-website/components/initiatives/data/initiatives.json:19]: this is causing a linting error, if we not using eventName anywhere we could remove it from everything else, and also from the type

**Review by andrew032011 — COMMENTED** [review]
Can you rerequest review from me once you fixed the type check error?

File: new-dti-website/components/initiatives/InitiativeDisplay.tsx (hunk 1 above)
Comment (Bookie0) [reply new-dti-website/components/initiatives/InitiativeDisplay.tsx:52]: LOL i accidentally edited smth oops

Comment (Bookie0) [reply new-dti-website/components/initiatives/InitiativeDisplay.tsx]: fixed [here](https://github.com/cornell-dti/idol/pull/787/commits/b67a197c6c1002475ecf9cc76f3ea5ea402ccdf0)

Comment (Bookie0) [reply new-dti-website/components/initiatives/InitiativeDisplay.tsx]: ok wait

File: new-dti-website/components/initiatives/data/initiatives.json (hunk 2 above)
Comment (Bookie0) [reply new-dti-website/components/initiatives/data/initiatives.json:19]: wait ok i messed up smth but i think i fixed it here/

File: new-dti-website/components/initiatives/InitiativeDisplay.tsx [hunk 3]
```diff
@@ -49,7 +46,7 @@ const Initiative = ({
           width >= LAPTOP_BREAKPOINT ? inter.className : ibm_plex_mono.className
         }`}
       >
-        {parts.map((part, index) => (part === eventName ? <b key={index}>{part}</b> : part))}
+{description}
       </p>
     </article>
   );
```
Comment (andrew032011) [inline new-dti-website/components/initiatives/InitiativeDisplay.tsx:52]: FYI `eventName` still appears below too (line 68 and 70), and type checker caught it

Comment (Bookie0) [reply new-dti-website/components/initiatives/InitiativeDisplay.tsx:45]: ohh good catch, should be fixed now i think...

**Review by andrew032011 — APPROVED** [review]

**Pull request merged**